### ISSUE TITLE
Прыгают иконки в трее при взаимодействиии в главном окне

### ISSUE THREAD

**Issue by Makishima** [BladeMight/Mahou#308]
Если каким-либо образом применить настройки (ОК/Применить) или закрыть окно, то иконки в трее перерисовываюсят и не созраняют свою позицию.

**Comment by BladeMight** [comment]
Тяжелый вопрос... Ты про NCS? Может просто не удалять их и создавать заново как сейчас? Т.е. если они уже существуют то оставить их, если ты их выбрал/удалил тогда убрать.

**Comment by Makishima** [comment]
Да, про них. Просто они перерисовываются, даже если ничего не менять в настройках.

**Comment by BladeMight** [comment]
@Makishima Сделал в bcfe8f0 чтобы:

> просто не удалять их и создавать заново как сейчас

Но насчёт порядка судя по [Raymond Chen's blog entry#99015](https://devblogs.microsoft.com/oldnewthing/20180614-00/?p=99015) - порядок иконок в трее неуправляемый, его нельзя программно изменить, система сама решает где и какой иконке быть. 

...Можно конечно нахально "влиять" на `Shell_TrayWnd` через приватные Window сообщения, но мне кажется так порядок других иконок может быть нарушен...

**Comment by Makishima** [comment]
В принципе порядок между перезапусками сохраняется. Так что если не заходить в настройки, то все ок.

**Comment by BladeMight** [comment]
Ну тогда на пока хватит.
